### PR TITLE
Text input, textarea, date input, and colour palette to #8a9198 

### PR DESCRIPTION
**Pull request by orinevares**

### CODE REVIEW

File: components/textarea/README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ form {
 .text_input {
   font-family: 'Noto Sans','Calibri', 'Arial', 'Sans Serif';
   font-size: 18px;
-  border: 2px solid #38598a;
+  border: 2px solid #8a9198;
   margin-top: 5px;
   margin-bottom: 15px;
   border-radius: 4;
```
Comment (dlevineBC) [inline components/textarea/README.md]: this is missing the px

File: components/date_input/README.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ form {
 
 .date_input_day, .date_input_month {
     height: 25px;
-    border: 2px solid #38598a;
+    border: 2px solid #8a9198;
     margin-top: 5px;
     margin-bottom: 15px;
     border-radius: 3px;
```
Comment (dlevineBC) [inline components/date_input/README.md]: Also update the border radius for all the date inputs to 4px

**Comment by dlevineBC** [comment]
@orinevares I think it also makes sense to update the border colours for the checkbox and radio buttons

**Comment by orinevares** [comment]
@dlevineBC All the edits have been made.

**Pull request merged**